sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -561,6 +561,45 @@ await foreach (ConfigurationSetting value in service.GetRevisionsAsync(selector,
             }
         }
 
+        [Test]
+        public async Task GetRevisionsByKeyAndLabel()
+        {
+            // The service keeps revision history even after the key was removed
+            // Avoid reusing ids
+            Recording.DisableIdReuse();
+
+            ConfigurationClient service = GetClient();
+            ConfigurationSetting testSetting = CreateSetting();
+
+            //Prepare environment
+            ConfigurationSetting setting = testSetting;
+
+            setting.Key = GenerateKeyId("key-");
+            ConfigurationSetting testSettingUpdate = setting.Clone();
+            testSettingUpdate.Label = "test_label_update";
+
+            try
+            {
+                await service.SetConfigurationSettingAsync(setting);
+                await service.SetConfigurationSettingAsync(testSettingUpdate);
+                AsyncPageable<ConfigurationSetting> revisions = service.GetRevisionsAsync(testSettingUpdate.Key, testSettingUpdate.Label, CancellationToken.None);
+
+                int resultsReturned = 0;
+                await foreach (ConfigurationSetting value in revisions)
+                {
+                    Assert.True(ConfigurationSettingEqualityComparer.Instance.Equals(value, testSettingUpdate));
+                    resultsReturned++;
+                }
+
+                Assert.AreEqual(1, resultsReturned);
```

Comment:
Do we have any tests that get multiple revisions?  It doesn't have to be for this overload, but if not, we should add one.

Reply:
`GetRevisions` test checks for several revisions.